deep_quoridor/src/main.py

```diff
@@ -1,15 +1,28 @@
 import argparse
 from arena_yaml_recorder import ArenaYAMLRecorder
 from arena import Arena
-from agents import SimpleAgent, RandomAgent
 from renderers import Renderer
+from agents import Agent
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Deep Quoridor")
     parser.add_argument("-N", "--board_size", type=int, default=None, help="Board Size")
     parser.add_argument("-W", "--max_walls", type=int, default=None, help="Max walls per player")
     parser.add_argument("-r", "--renderer", choices=Renderer.names(), default="results", help="Render mode")
     parser.add_argument("--step_rewards", action="store_true", default=False, help="Enable step rewards")
+    parser.add_argument(
+        "-p",
+        "--players",
+        type=str,
```

Comment:
argparse supports "list" arguments natively. Can you try changing it to that, if I"m not crazy? It will all be much cleaner, having the proper type from the get go